dtreeviz/models/shadow_decision_tree.py

```diff
@@ -390,14 +392,7 @@ def get_leaf_sample_counts_by_class(self):
 
     def _normalize_class_names(self, class_names):
```

Comment:
Could we remove this private function and use the one define in utils ?

Reply:
Oh right.  if I have moved to it then yes, let's remove :)